addon/globalPlugins/openLinkWith/__init__.py

```diff
@@ -117,6 +188,51 @@ def script_displayLinksInLastSpokenText(self, gesture):
 			DIALOG= MyDialog(gui.mainFrame, list_, browsers)
 			DIALOG.postInit()
 
+	@script(
+		# Translators: Message to be displayed in input help mode.
+		description= _("If standing on a link, display a menu "
+		"to give you the option to open the link with other browsers on  your machine."
+		),
+		gesture= "kb:alt+/"
```

Comment:
It’s really a good feature, the Alt+/ gesture is convenient enough, but this gesture does not include the NVDA key, will it conflict with other software? Although I can't think of an exact example right now.

Reply:
> It’s really a good feature, the Alt+/ gesture is convenient enough, but this gesture does not include the NVDA key, will it conflict with other software? Although I can't think of an exact example right now.

Thanks @cary-rowen for review
About the gesture, it is intended to resemble the context menu, so no need to NVDA key here, and when using it I feel it is near or like the context menu, for this I think it is appropriate. Moreover the user at any time, supposing this gesture has a role in some systems or regions, the user should learn to remove it through input gestures dialog, and assign for this purpose the gesture he wishes, so really I think the issue of the gesture should not be of big deal.
regards

Reply:
OK, everything looks fine, we can merge it.